doc/userdoc/guides/spatially_structured_networks.rst

```diff
@@ -986,15 +996,25 @@ Cut-off Gaussian
     :end-before: #{ end #}
 
 2D Gaussian
-   We conclude with an example using a two-dimensional Gaussian
-   distribution, i.e., a Gaussian with different widths in :math:`x`- and
-   :math:`y`- directions. This probability depends on displacement, not
-   only on distance:
+   Here we use a two-dimensional Gaussian distribution, i.e., a Gaussian with
+   different widths in :math:`x`- and :math:`y`- directions. This probability
+   depends on the distance along the coordinates axes, not the distance in the
+   plane:
```

Comment:
I am not sure what you mean here. In the original documentation, "displacement" means (delta_x, delta_y), while "distance" means the euclidean length of that vector.

Reply:
If I understand the code correctly, calling, e.g., ```nest.spatial.distance.x``` results [this](https://github.com/nest/nest-simulator/blob/6adfd2cbb2df87123453cd2b46ef1aee40837d40/nestkernel/parameter.cpp#L172) function being called. In this case, this should result in the following return:
```
return std::abs( layer.compute_displacement( source_pos, target_pos, dimension_ - 1 ) );
```
What I wanted to express is that this function does not return the displacement (delta_x, delta_y) but (\abs{delta_x}, \abs{delta_y}), i.e. the L^1 norm (which tried to describe by "distance along the coordinate axes"). 
For this reason I also have to pass the following connection dictionary when using the Gabor function
```
conndict = {'rule': 'pairwise_bernoulli',
            'p': nest.spatial_distributions.gabor(
                nest.spatial.target_pos.x - nest.spatial.source_pos.x,  # only this yields displacement along x axis
                nest.spatial.target_pos.y - nest.spatial.source_pos.y,  # only this yields displacement along y axis
)}
```

Reply:
Sorry for the diversion in the above comment.
I know understand my confusion and you are absolutely right: Of course also ```nest.spatial_distributions.gaussian2D``` depends on the displacement and and only in special cases on the ```nest.spatial.distance.{x,y}```.
I reverted the introduced mistake (9a0a216).
Maybe one could change the above cited example for the connection dictionary for the 2D Gaussian since it explicitly mentions that the probability depends on the displacement, but then uses the distance along the axes.